recipes/newrelic/memcached/debian.yml

```diff
@@ -0,0 +1,110 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: memcached-open-source-integration
+displayName: Memcached Open Source Integration
+description: New Relic install recipe for default Memcached Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-memcached
+
+installTargets:
+  - type: host
+    os: linux
+    platform: "debian"
+  - type: host
+    os: linux
+    platform: "ubuntu"
+
+# keyword convention for dealing with search terms that could land someone on this instrumentation project
+keywords:
+  - Infrastructure
+  - Integration
+  - memcached
+
+# Examine Infrastructure events for correlated data
+processMatch:
+  - memcached
+  - memcache
+
+# Matches partial list of the Log forwarding parameters
+logMatch:
+  - name: memcached
+    file: /var/log/memcached.log
+
+# NRQL the newrelic-cli will use to validate the agent/integration this recipe
+# installed is successfully sending data to New Relic
+validationNrql: "SELECT count(*) from MemcachedSample where hostname like '{{.HOSTNAME}}%' FACET entityGuid SINCE 10 minutes ago"
+
+# Prompts for input from the user. These variables then become
+# available to go-task in the form of {{.VAR_NAME}}
+inputVars:
+  - name: "NR_CLI_USERNAME"
+    prompt: "SASL Username (if applicable)"
+  - name: "NR_CLI_PASSWORD"
+    prompt: "SASL Password (if applicable)"
+    secret: true
```

Comment:
@JakubKotkowiak Since these are optional, we should check to see if these are empty (and if so, omit them from `memcached-config.yml` altogether).